### ISSUE TITLE
Grub (?) hangs after installing inux-firmware-image-4.2.8ph+

### ISSUE THREAD

**Issue by polentino** [raphael/linux-samus#68]
Hi, today I downloaded Mint 17.3 and performed a fresh install on my Chromebook LS.
I played for half an hour with it, and then I decided to install *inux-firmware-image-4.2.8ph+_4.2.8ph+-1_amd64.deb* & friends to make the touchpad, audio and keyboard to work properly.
I cloned the repo as described in your guide, installed all the deb files inside *linux-samus/build/debian* but, once I rebooted, I couldn't log in anymore.
In short, I get to see SeaBIOS menu but, when I hit Enter in order to boot from my ssd drive, the screen hangs and doesn't respond anymore, no matter how long I wait.

**Comment by raphael** [comment]
This issue has been reported before but I don't think it was resolved. I am not using a debian setup so it's difficult for me to help. Closing as duplicate of #54.
